models/general/knowledge/en/who_is.txt

```diff
@@ -0,0 +1,11 @@
+# answers who is question efficiently
```

Comment:
Hey, can you take a look at https://products.wolframalpha.com/api/explorer/?
This gives a much better response for who+is or what+is? Try that maybe?